src/main/java/org/apache/commons/daemon/DaemonInitException.java

```diff
@@ -28,16 +28,29 @@
     // don't rely on Throwable#getCause (jdk1.4)
     private final Throwable cause;
 
+    /**
+     * Constructor for DaemonInitException.
+     * @param message Message.
+     */
     public DaemonInitException(String message) {
         super(message);
         this.cause = null;
     }
 
+    /**
+     * Constructor for DaemonInitException.
+     * @param message Message.
+     * @param cause cause.
+     */
     public DaemonInitException(String message, Throwable cause) {
         super(message);
         this.cause = cause;
     }
 
+    /**
+     * Method for getting cause.
```

Comment:
Javadocs should use the active voice, for getters, use "Gets..."; for setters "Sets..." and so on.